php1/mysql/login/login.php

```diff
@@ -0,0 +1,28 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Login</title>
+  <link rel="stylesheet" type="text/css" href="../../css/style1.css">
+</head>
+<body>
+  <?php require("loginvalidate.php"); ?>
```

Comment:
According to coding convention require should be in single qoute and remove brackets like `require 'loginvalidate.php'`